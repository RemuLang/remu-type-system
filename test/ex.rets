store 0 = '0;
store 1 = '1;
store 2 = '2;
'0 = ^0;
'1 = forall {a, b} [a, [b, b]];
'2 = {x : ^0, b: ^1};
