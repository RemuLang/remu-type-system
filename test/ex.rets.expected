0 ^0;
1 ['3, ['4, '4]];
2 {x: ^0, b: ^1};
